sympy/plotting/plot.py

```diff
@@ -1577,91 +1577,89 @@ def plot(*args, **kwargs):
+    Parameters
+    ==========
 
-    ``plot_parametric((expr_x, expr_y, range), ..., **kwargs)``
+    args
+        Common specifications are:
 
-    Range has to be specified for every expression.
+        - Plotting a single parametric curve with a range
+            ``plot_parametric((expr_x, expr_y), range)``
+        - Plotting multiple parametric curves with the same range
+            ``plot_parametric((expr_x, expr_y), ..., range)``
+        - Plotting multiple parametric curves with different ranges
+            ``plot_parametric((expr_x, expr_y, range), ...)``
 
-    Default range may change in the future if a more advanced default range
-    detection algorithm is implemented.
+        ``expr_x`` is the expression representing $x$ component of the
+        parametric function.
 
-    Arguments
-    =========
+        ``expr_y`` is the expression representing $y$ component of the
+        parametric function.
 
-    ``expr_x`` : Expression representing the function along x.
+        ``range`` is a 3-tuple denoting the parameter symbol, start and
+        stop. For example, ``(u, 0, 5)``.
 
-    ``expr_y`` : Expression representing the function along y.
+        If the range is not specified, then a default range of (-10, 10)
+        is used.
 
-    ``range``: (u, 0, 5), A 3-tuple denoting the range of the parameter
-    variable.
+        However, if the arguments are specified as
+        ``(expr_x, expr_y, range), ...``, you must specify the ranges
+        for each expressions manually.
 
-    Keyword Arguments
-    =================
+        Default range may change in the future if a more advanced
+        algorithm is implemented.
 
-    Arguments for ``Parametric2DLineSeries`` class:
+    adaptive : bool, optional
+        Specifies whether to use the adaptive sampling or not.
 
-    ``adaptive``: Boolean. The default value is set to True. Set adaptive to
-    False and specify ``nb_of_points`` if uniform sampling is required.
+        The default value is set to ``True``. Set adaptive to ``False``
+        and specify ``nb_of_points`` if uniform sampling is required.
 
-    ``depth``: int Recursion depth of the adaptive algorithm. A depth of
-    value ``n`` samples a maximum of `2^{n}` points.
+    depth :  int, optional
+        The recursion depth of the adaptive algorithm. A depth of
+        value $n$ samples a maximum of $2^n$ points.
 
-    ``nb_of_points``: int. Used when the ``adaptive`` is set to False. The
-    function is uniformly sampled at ``nb_of_points`` number of points.
+    nb_of_points : int, optional
+        Used when the ``adaptive`` flag is set to ``False``.
 
-    Aesthetics
-    ----------
+        Specifies the numher of the points used for the uniform
+        sampling.
 
-    ``line_color``: function which returns a float. Specifies the color for the
-    plot. See ``sympy.plotting.Plot`` for more details.
+    line_color : function
+        A function which returns a float.
```

Comment:
I like the breadcrumb telling you where to look to learn about what the float should be.